automated install/basic-install.sh

```diff
@@ -1810,6 +1811,16 @@ installPiholeWeb() {
     printf "%b  %b %s\\n" "${OVER}" "${TICK}" "${str}"
 }
 
+# Creates custom DNS file if it does not exist
```

Comment:
Instead of creating a function for this one task you should add it to the `installConfigs()` section. Something similar to https://github.com/pi-hole/pi-hole/blob/331502e14cc31972f4653b9741b931be7569031a/automated%20install/basic-install.sh#L1475

Reply:
Done. Thanks for the hint, looks much cleaner now.